### ISSUE TITLE
Can't create accurate time entry for 11:15am - 1:00pm

### ISSUE THREAD

**Issue by chrisfosterelli** [tickbin/tickbin#197]
![image](https://cloud.githubusercontent.com/assets/2072672/15342101/c328452e-1c47-11e6-9d53-a80cf830e4e9.png)

Date is correct, interpreted start and end time is not although I was quite specific. The first three attempts failed, the only way to get the entry into tickbin appears to be using the military parser.

@tickbin/core Can anyone else reproduce this? Probably should fix ASAP if so.

**Comment by chrisfosterelli** [comment]
Also seems to have issue with this:

**Comment by chrisfosterelli** [comment]
This is likely related to the fact that the word "Get" is in there, which is [a timezone](http://www.timeanddate.com/time/zones/get). It's marking GET as part of the time, not the message.

**Comment by chadfawcett** [comment]
I was able to reproduce. I think you are right with your assumptions on the "Get" string in the message. "Get" is being stripped out of the message.

![screen shot 2016-05-17 at 4 25 28 pm](https://cloud.githubusercontent.com/assets/3250463/15342646/f94707d6-1c4b-11e6-8a78-9715e8ffbbfd.png)

**Comment by jonotron** [comment]
This should be fixable by manually specifying the parsers and refiners that we need rather than using the defaults provided by chrono. We could not include this refiner: https://github.com/wanasit/chrono/blob/master/src/refiners/ExtractTimezoneAbbrRefiner.js which extracts the TZ from the string. 

We should be thoughtful as to how we do this tho, as it may be useful to allow timezone specifying (for people who travel a lot).

A couple ideas:

* add a string `--tz=GET` flag to allow manually specifying the timezone of the entry.
* add a boolean `--tz` flag that sets up the parser to include the timezone refiner so one could enter "1-2pm GET worked on stuff"
* write our own refiner to require timezones be in UPPERCASE (this feels wrong as I usually write GET in uppercase when referring to HTTP requests)

**Comment by jonotron** [comment]
Another thought: we could make tickbin a bit more interactive and detect TZs in the entry and then ask the user if they want us to treat something like "GET" as a timezone or not.

**Comment by chrisfosterelli** [comment]
I like the idea of `--tz=GET`. This would make it not possible to specify a different starting and stopping timezone, but that'd be an exceptionally rare case.

**Comment by jonotron** [comment]
@chrisfosterelli maybe if I was on a flight and I was like "7am PST - 2pm HAST fixing bugs on flight to Hawaii"

**Comment by chrisfosterelli** [comment]
@jonotron Right, but that won't work if we get rid of string parsing and use `--tz=GET`, which I think would be the most natural option.

**Comment by jonotron** [comment]
I kinda like the `--tz` to turn on timezone parsing... also, it's something that we could easily make an option in `.tickbinrc`
